public/js/p3/widget/DownloadTooltipDialog.js

```diff
@@ -55,10 +56,19 @@ define([
 			console.log("DOWNLOAD TYPE: ", type)
 			if(conf.generateDownloadFromStore && this.grid && this.grid.store && type && this["_to" + type]){
 				var query = "in(" + pkField + ",(" + sel.join(",") + "))&sort(+" + pkField + ")&limit(2500000)"
+				var field = pkField;
+				var selectedSubsystems = selection;
```

Comment:
We don't need to introduce new variables.
1. you can use pkField in line 63.
2. you can use selection in line 64.